conans/client/conf/__init__.py

```diff
@@ -54,7 +54,7 @@
                     "13.0", "13.1", "13.2", "13.3", "13.4", "13.5", "13.6",
                     "14.0", "14.1", "14.2", "14.3", "14.4"]
         sdk_version: [null, "10.13", "10.14", "10.15", "11.0", "11.1", "11.3", "12.0", "12.1",
-                        "12.3", "13.0", "13.1", "13.3", "14.0"]
+                        "12.3", "13.0", "13.1", "13.3", "14.0", "14.5"]
```

Comment:
shouldnt we add some intermediate versions too? https://xcodereleases.com/

Reply:
Added intermediate versions!
Missing apple vision and others, will update in a patch